src/core/qgsbookmarkmanager.h

```diff
@@ -96,6 +96,19 @@ class CORE_EXPORT QgsBookmark
      */
     void setExtent( const QgsReferencedRectangle &extent );
 
+
+    /**
+     * Returns the bookmark's map rotation.
+     * \see setRotation()
```

Comment:
You'll need to add a \since QGIS X.XX version to both this and the setRotation function so the documentation can reflex the minimum version required to use these.

Reply:
Ok